src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -26,6 +26,8 @@
 final class OpenApiNormalizer implements NormalizerInterface, CacheableSupportsMethodInterface
 {
     public const FORMAT = 'json';
+    public const JSONOPENAPI_FORMAT = 'jsonopenapi';
```

Comment:
```suggestion
    public const JSON_FORMAT = 'jsonopenapi';
```

(`OPENAPI` is redundant)